src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyContent.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import PermissionAlert from 'app/views/settings/project/permissionAlert';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import Form from 'app/views/settings/components/forms/form';
+import {fields} from 'app/data/forms/projectGeneralSettings';
+import AsyncView from 'app/views/asyncView';
+import ProjectActions from 'app/actions/projectActions';
+import {Scope} from 'app/types';
+
+import {DataPrivacy} from './types';
+
+class ProjectDataPrivacyContent extends AsyncView<DataPrivacy> {
+  static contextTypes = {
+    organization: PropTypes.object.isRequired,
+    router: PropTypes.object,
+  };
+
+  getEndpoints() {
+    const {orgId, projectId} = this.props.params;
+    const endpoints = [['data', `/projects/${orgId}/${projectId}/`]];
+    return endpoints as Array<[string, string]>;
+  }
+
+  renderBody() {
+    const {organization} = this.context;
+    const project = this.state.data;
+    const {orgId, projectId} = this.props.params;
+    const endpoint = `/projects/${orgId}/${projectId}/`;
+    const access = new Set(organization.access);
```

Comment:
Can we utilize `src/sentry/static/sentry/app/components/acl/access.tsx`?

Reply:
Honestly..I don't know. I would need to investigate